core/model_induction.py

```diff
@@ -170,12 +170,23 @@ def dump(self, file_path):
             file.write(file_buffer)
 
 
-def train_decision_tree(train_features, train_label):
+def train_custom_tree(train_features, train_label):
     model = CustomBinaryClassifier()
     model.fit(train_features, train_label)
     return model
 
-def train_classifier_tree(train_features, train_label):
-    model = RandomForestClassifier(n_estimators=30, max_depth=40)
-    model.fit(*balance_binary_dataset(train_features, train_label, skew_false=8))
+def train_decision_tree(train_features, train_label):
+    model = DecisionTreeClassifier()
+    model.fit(train_features, train_label)
+    return model
+
+def train_classifier_tree(train_features, train_label, **kwargs):
+    estimators_key, estimators_default = 'n_estimators', 30
+    depth_key, depth_default = 'max_depth', 40
+
+    kwargs[estimators_key] = kwargs[estimators_key] if estimators_key in kwargs else estimators_default
+    kwargs[depth_key] = kwargs[depth_key] if depth_key in kwargs else depth_default
```

Comment:
any benefits of doing kwargs manip here vs just defining default args in function header?

Reply:
That would be the sane way to do it.